plugins/inputs/http_listener_v2/README.md

```diff
@@ -18,8 +18,12 @@ This is a sample configuration for the plugin.
   ## Address and port to host HTTP listener on
   service_address = ":8080"
 
-  ## Path to listen to.
-  # path = "/telegraf"
+  ## Paths to listen to.
+  # paths = ["/telegraf"]
```

Comment:
This is a breaking change, but we need to preserve backward compatibility for existing configurations. Thus, you need to keep the `path` option (maybe mark deprecated).

Reply:
I simply appends `path` to `paths` to keep backward compability